docs/customising/make_a_new_theme.md

```diff
@@ -0,0 +1,258 @@
+    (which will probably be GitHub) whenever you have any changes to deploy.
+ </p>
+</div>
+
+## 6. See the new colour!
+
+Now if you look at the home page on your development server, you'll
+see the new colour.
+
+<div class="attention-box helpful-hint">
+  If you change the colour again, and it doesn't update when you refresh the
+  browser, you <em>might</em> need to delete everything in 
+  <code>/tmp/cache</code> within your installation to force this
+  behaviour.
+</div>
+
+## 7. Change the logo
+
+To change the default, placeholder logo, replace the file
+`/assets/images/logo.png` with one of your own. We recommend you keep to the
+same aspect ratio to start with, because that will fit in with the rest of the
+layout. Of course, you can change all of that later: this is just to get you
+comfortable with the most important changes.
+
+Alaveteli uses Rails'
+[asset pipeline](http://guides.rubyonrails.org/asset_pipeline.html),
+and when it gathers its resources together (of which your `logo.png` is one),
+anything in your custom theme will override what's in the default theme. So
+your logo will appear if you put it into your theme, because you've nominated
+your theme as the primary one in `THEME_URLS`. Any resources that aren't in
+your theme will be taken from the core asset directories instead.
+
+## 8. Commit the changes
+
+<div class="attention-box info">
+  If you're used to working with git, you might have done this already, when
+  you changed the colour and the logo.
+</div>
+
+You need to commit the changes (that is, tell git what you changed and why),
+and then push those changes back up to your repo on GitHub. This is because,
+eventually, your production site will be pulling the theme from there (because
+you'll put its URL in your THEME_URL config setting for your production server,
+just like you have done for this development one).
+
+<div class="attention-box helpful-hint">
+  Note that <em>inside</em> your Vagrant virtual machine, you probably
+  won't be able to access GitHub with your user settings, and so on.
+  So you may need to do this from "outside" the VM. This can trip you
+  up because the command shell looks very similar when you are inside
+  a <code>vagrant&nbsp;ssh</code> session, and when you are not.
+</div>
+
+Well done — when you've customised your theme and pushed that work back up to
+GitHub, you're well on your way to having your own Alaveteli site.
+
+Note another reason for pushing to GitHub is that we can look at it (or, more
+usefully perhaps, run it on a development server of our own by putting that URL
+into our THEME_URL setting) if you need any help.
+
+<div class="attention-box warning">
+  The <code>rake theme:install</code> task will checkout your theme in with a
+  <em>detached&nbsp;HEAD</em>. This is correct, but it might catch you out if
+  you then try to do any more development work in there. You need to checkout a
+  specific branch if you want to do more customising work.
```

Comment:
I think this goes back to having a stronger pointer on to how to set up a theme for development as opposed to a quick test/demo